common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java

```diff
@@ -95,7 +106,25 @@ public Row selectRow() {
         return Graphene.createPageFragment(Row.class, row);
     }
 
+    /**
+     * @param exactRowText decides whether the row will be selected by its exact label (equals) or by part of its label (contains)
+     */
+    public Row selectRow(Boolean exactRowText) {
+        WebElement row = navigate(exactRowText)[1];
+        if (row == null) {
+            throw new IllegalStateException("No address for selecting a row given.");
+        }
+        return Graphene.createPageFragment(Row.class, row);
+    }
+
     private WebElement[] navigate() {
+        return navigate(false);
+    }
+
+    /**
```

Comment:
Please remove this duplicate javadoc. It's not necessary in private method. Everything else seems OK for me.